io/router/router.go

```diff
@@ -102,6 +102,12 @@ func (q *Router) TextInputState() TextInputState {
 	return q.kqueue.InputState()
 }
 
+// TextInputMode returns the input state from the most recent
+// call to Frame.
```

Comment:
Is it better to say

// TexInputMode returns the input mode from the most recent key.InputOp.

? 

Then there won't be a sudden switch to KeyboardDefault just because a Frame contains no Editors.

Remember to change the comment for (*keyQueue).InputMode as well.